src/layouts/account.tsx

```diff
@@ -9,13 +9,15 @@ import { accountSearchItem } from "@/constants/search";
 
 import { SearchProvider } from "@/providers/search";
 
+import usePageTracking from "@/hooks/use-page-tracking";
+
 import AccountSidebar from "@/components/account-sidebar";
 import AppNavbar from "@/components/app-navbar";
 
 export default function AccountLayout() {
   // Persisting the sidebar state in the cookie.
   const defaultOpen = Cookies.get("sidebar_state") !== "false";
-
+  usePageTracking();
```

Comment:
@ren3459 why we use this only for account layout? As far as I understand we should use it for every page (main tsx?).